packages/create-cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "create-checkly",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
I'll release a new version of `create-checkly` after this. Some small changes were made within that package.